docs/ssh_tunnel.md

```diff
@@ -0,0 +1,276 @@
+    ```
+
+
+3. **On the remote virtual machine**, copy the content of this public key file and paste it into the authorized_keys file, for instance editing it with `nano`
+
+    ```
+    cloud_machine$ nano /home/tunnel/.ssh/authorized_keys
+    ```
+
+
+4. On the DAQ instance machine, start the new ssh-tunnel service
+
+    ```
+    daq_machine$ systemctl start ssh-tunnel.service
+    ```
+
+
+5. Check that the new ssh-tunnel service is running on the DAQ instance machine 
+
+    ```
+    daq_machine$ systemctl status ssh-tunnel.service 
+
+
+    ● digital-building.service - AutoSSH tunnel service
+       Loaded: loaded (/etc/systemd/system/ssh-tunnel.service; enabled; vendor preset: enabled)
+       Active: active (running) since Thu 2020-04-23 14:14:41 BST; 27s ago
+     Main PID: 3316 (autossh)
+        Tasks: 2 (limit: 4915)
+       CGroup: /system.slice/ssh-tunnel.service
+               ├─3316 /usr/lib/autossh/autossh -v -M 0 -N -R 19991:localhost:22 -o ServerAliveInterval 60 -o ServerAliveCountMax 3 -o StrictHostKeyChecking=no -o BatchMode=yes -i /home/tunnel/.ssh/id_rsa tunnel@34.66.115.220
+               └─3319 /usr/bin/ssh -v -N -R 19991:localhost:22 -o ServerAliveInterval 60 -o ServerAliveCountMax 3 -o StrictHostKeyChecking=no -o BatchMode=yes -i /home/tunnel/.ssh/id_rsa tunnel@digital-building.org
+
+    Apr 23 14:14:41 daq_machine autossh[3316]: Authenticated to digital-building.org ([130.211.66.54]:22).
+    Apr 23 14:14:41 daq_machine autossh[3316]: debug1: Remote connections from LOCALHOST:19991 forwarded to local address localhost:22
+    Apr 23 14:14:41 daq_machine autossh[3316]: debug1: Requesting no-more-sessions@openssh.com
+    Apr 23 14:14:41 daq_machine autossh[3316]: debug1: Entering interactive session.
+    Apr 23 14:14:41 daq_machine autossh[3316]: debug1: pledge: network
+    Apr 23 14:14:42 daq_machine autossh[3316]: debug1: client_input_global_request: rtype hostkeys-00@openssh.com want_reply 0
+    Apr 23 14:14:42 daq_machine autossh[3316]: debug1: Remote: /home/tunnel/.ssh/authorized_keys:8: key options: agent-forwarding port-forwarding pty user-rc x11-forwarding
+    Apr 23 14:14:42 daq_machine autossh[3316]: debug1: Remote: /home/tunnel/.ssh/authorized_keys:8: key options: agent-forwarding port-forwarding pty user-rc x11-forwarding
+    Apr 23 14:14:42 daq_machine autossh[3316]: debug1: remote forward success for: listen 19991, connect localhost:22
+    Apr 23 14:14:42 daq_machine autossh[3316]: debug1: All remote forwarding requests processed
+    ```
+
+# Achieving Remote Access
+
+The final step is to login to the cloud virtual machine and check that you are able to connect to the DAQ instance machine through the ssh tunnel.
```

Comment:
break lines

Reply:
Thanks for checking, I've now broken the lines as suggested.